week_3/2018202181SZH/src/unit.h

```diff
@@ -0,0 +1,26 @@
+#ifndef UNIT_H
+#define UNIT_H
+#include<iostream>
+#include<string>
+#include<set>
+#include<queue>
+ 
+void input_first_url(std::string &url);
+class website {
+    private:

```

Comment:
No problem here but this line `private:` could be omitted since it's the default.

Reply:
> No problem here but this line `private:` could be omitted since it's the default.

Just a notice. You can still write like this if you like.